File: app/src/main/java/com/monster/app/niceweather/models/ForecastModel.java
package com.monster.app.niceweather.models;

import java.util.List;

/**
 * Created by monster on 2/24/18.
 */

public class ForecastModel {
    public TempModel temp;
    public List<ForecastCityModel.WeatherModel> weather;

    public static class TempModel {
        public double day;
        public double min;
        public double max;
        public double night;
        public double eve;
        public double morn;
    }
}

File: app/src/main/java/com/monster/app/niceweather/models/ForecastModel.kt
package com.monster.app.niceweather.models

/**
 * Created by monster on 2/24/18.
 */

class ForecastModel(val temp: TempModel, val weather: List<ForecastCityModel.WeatherModel>) {

    class TempModel (
        val day:Double = 0.0,
        val min:Double = 0.0,
        val max:Double = 0.0,
        val night:Double = 0.0,
        val eve:Double = 0.0,
        val morn:Double = 0.0
    )
}
